### ISSUE TITLE
Issue with Validtor 

### ISSUE THREAD

**Issue by Rashmi-sb** [CMSgov/price-transparency-guide-validator#41]
Has anyone used this validator, we tested it and we see some of required the JSON entities names are not validated against correct schema entities name
Eg : Billing_code_type(schema naming ) is not validated to BillingCodeType(incorrect naming) , even if we remove this whole element from the output JSON , the validator says Valid JSON .

**Comment by Orange-1234** [comment]
So far it seems to be working for us.  I have used the validator tool on a subset of our data for the In Network rates to catch issues before creating the entire file.
It caught: 
1)Had JSON entity name "expriration_date" instead of "expiration_date"  gave error that "expiration_date" was missing.  
2) Service Code: Single digit service codes (i.e. 1 - 9) were not padded with a zero - gave error that invalid values were included. 3) Had JSON entity name of "reporting_type" instead of reporting_entity_type - gave error message that "reporting_entity_type" was missing.

I could not reproduce your issue.  I took a valid JSON file & did a global replace to make "billing_code_type" be "billingcodetype" - it said the file was invalid said "billing_code_type" was missing.

FYI - We are using provider references and the subset of data I was using has ~ 900 "In Network Objects" and represents ~37,000 negotiated rates.
